lib/experimental/modules/class-gutenberg-modules.php

```diff
@@ -90,27 +116,25 @@ public static function print_import_map() {
 	 * Prints all the enqueued modules using <script type="module">.
 	 */
 	public static function print_enqueued_modules() {
-		foreach ( self::$enqueued as $module_identifier ) {
-			if ( isset( self::$registered[ $module_identifier ] ) ) {
-				$module = self::$registered[ $module_identifier ];
-				wp_print_script_tag(
-					array(
-						'type' => 'module',
-						'src'  => $module['src'] . self::get_version_query_string( $module['version'] ),
-						'id'   => $module_identifier,
-					)
-				);
-			}
+		foreach ( self::get_enqueued() as $module_identifier => $module ) {
+			wp_print_script_tag(
+				array(
+					'type' => 'module',
+					'src'  => $module['src'] . self::get_version_query_string( $module['version'] ),
+					'id'   => $module_identifier,
+				)
+			);
 		}
 	}
 
 	/**
-	 * Prints the link tag with rel="modulepreload" for all the static
-	 * dependencies of the enqueued modules.
+	 * Prints the the static dependencies of the enqueued modules using <link rel="modulepreload">.
```

Comment:
similar note here as above. we might use something like "using a link tag with the `modulepreload` value for its `rel` attribute."

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57231/commits/5120b7d213304799b611addb94adf6389d80c362.